nextflow/accession.nf

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env nextflow
+
+def helpMessage() {
+    log.info"""
+    Accession and ingest variant files.
+
+    Inputs:
+            --accession_props       properties files for accessioning
+            --project_accession     project accession
+            --instance_id           instance id to run accessioning
+    """
+}
+
+params.accession_props = null
+params.project_accession = null
+params.instance_id = null
+// executables
+params.executable = ["bgzip": "bgzip", "tabix": "tabix", "copy_to_ftp": "copy_to_ftp"]
+// java jars
+params.jar = ["accession_pipeline": "accession_pipeline"]
+// help
+params.help = null
+
+// Show help message
+if (params.help) exit 0, helpMessage()
+
+// Test input files
+if (!params.accession_props || !params.project_accession || !params.instance_id) {
+    if (!params.accession_props)    log.warn('Provide an accessions properties file using --accession_props')
+    if (!params.project_accession)  log.warn('Provide a project accession using --project_accession')
+    if (!params.instance_id)        log.warn('Provide an instance id using --instance_id')
+    exit 1, helpMessage()
+}
+
+accession_props = Channel.fromPath(params.accession_props)
+
+
+/*
+* Accession VCF
+*/
+process accession_vcf {
+    clusterOptions '-g /accession/instance-$params.instance_id'
+
+    input:
+        path "accession.properties" from accession_props
+
+    output:
+        path "00_logs/accessioning.*.log" into accessioning_log
+        path "00_logs/accessioning.*.err" into accessioning_err
+        path "60_eva_public/*.vcf" into accessioned_vcfs
```

Comment:
This comment applies to many steps in the workflows:
In Nextflow, the script is being executed in a temp directory placed in the current directory which usually looks like this: `work/91/eff364f4a21f7a5df088022564c7bd`. 
The output that you declare above will be search in this temp directory and when not found, the step will fail.
I think the best way would be to get the `00_logs` and `60_eva_public` as input so they are linked in the temp directory, then you can have the files as an output. 
This said it mean you can't use the wildcards because you can't guarantee that there won't be any other VCF/logs in those folders.

Reply:
This and the other input/output issues should be fixed, please have a look and let me know.